diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..6b950f9 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -75,6 +75,7 @@ if(${RNS_NEW_ARCH_ENABLED})
 else()
     target_link_libraries(rnscreens
         ReactAndroid::jsi
+        c++_shared
         android
     )
 endif()
